webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -31,8 +31,13 @@
         class="general-actions left-pane"
         *ngIf="currentTab === 'reports' && !selectMode"
         [ngClass]="{ 'col-sm-3': sidebarFilter?.isOpen, 'col-sm-4': !sidebarFilter?.isOpen }">
-        <div class="actions dropup" dropdown [dropup]="true" [isDisabled]="!nonContactForms?.length">
-          <a class="mm-icon mm-icon-inverse mm-icon-caption dropdown-toggle" dropdownToggle dropdata-toggle="dropdown" [ngClass]="{'mm-icon-disabled': !nonContactForms?.length}">
+        <div
+          class="actions dropup"
+          dropdown [dropup]="true"
+          [isDisabled]="!nonContactForms?.length"
+          mmAuth [mmAuthAny]="['can_edit', 'can_export_all', 'can_export_messages']"
```

Comment:
Once we implement the FAB, we'll have a better solution to declare and render action buttons, we'll have a better way to know if the container has any button visible and the html will be cleaner.

Reply:
I don't think this behaves quite like you want, because the mmAuthAny directive will return true if the user has any of the permissions, so adding `can_edit` doesn't hide it.

I _think_ this may work..

```suggestion
          mmAuth="can_edit" [mmAuthAny]="['can_export_all', 'can_export_messages']"
```

Reply:
The export permissions aren't modifying the `medic` db, I've tested with online/offline users and they can export successfully. 
I think export permissions aren't depending on `can_edit`

Reply:
There are other conditions we should consider when hiding the containers of action buttons, if we want to make it perfect for every corner case. 
For example: if it's mobile, if it's online user, if it has the data the action needs (phone number), etc...

We can do this a lot easier once we do the FAB work with a better design to declare and render action buttons, the container should easily know if it's displaying actions buttons. 

So we can:
1. Leave it like this a bit imperfect
2. Let the container display always, regardless if it has buttons or not. 

![Screen Shot 2022-08-24 at 3 50 59 pm](https://user-images.githubusercontent.com/66472237/186374896-b9f9f280-4598-46fc-b013-0f35c8eeb1e3.png)

Reply:
> Let the container display always, regardless if it has buttons or not.

I think that's fine for now. You're right, it's complicated to work out whether the bar should show or not. It's a bit of an edge case to have no buttons at all, so let's leave this until we do the FAB redesign.